apps/meteor/client/components/Sidebar/SidebarItemsAssembler.tsx

```diff
@@ -12,19 +13,27 @@ type SidebarItemsAssemblerProps = {
 
 const SidebarItemsAssembler: FC<SidebarItemsAssemblerProps> = ({ items, currentPath }) => {
 	const t = useTranslation();
+
 	return (
 		<>
-			{items.map(({ href, pathSection, i18nLabel, name, icon, permissionGranted, pathGroup, tag }) => (
-				<SidebarNavigationItem
-					permissionGranted={permissionGranted}
-					pathGroup={pathGroup || ''}
-					pathSection={href || pathSection || ''}
-					icon={icon}
-					label={t((i18nLabel || name) as Parameters<typeof t>[0])}
-					key={i18nLabel || name}
-					currentPath={currentPath}
-					tag={t.has(tag as Parameters<typeof t>[0]) ? t(tag as Parameters<typeof t>[0]) : undefined}
-				/>
+			{items.map(({ href, pathSection, i18nLabel, name, icon, permissionGranted, pathGroup, tag, divider, externalUrl }) => (
+				<Box key={i18nLabel || name}>
+					{divider && (
+						<Box mi='24px'>
+							<Divider />
+						</Box>
+					)}
+					<SidebarNavigationItem
+						permissionGranted={permissionGranted}
+						pathGroup={pathGroup || ''}
+						pathSection={href || pathSection || ''}
+						icon={icon}
+						label={t((i18nLabel || name) as Parameters<typeof t>[0])}
+						currentPath={currentPath}
+						tag={t.has(tag as Parameters<typeof t>[0]) ? t(tag as Parameters<typeof t>[0]) : undefined}
```

Comment:
Can a type guard be used here? Something like 
`isTranslationKey = (v: any): v is TranslationKey => t.has(v)`

Reply:
Solved